blamer.el

```diff
@@ -992,6 +992,12 @@ LOCAL-TYPE is force replacement of current `blamer-type' for handle rendering."
   (remove-hook 'post-command-hook #'blamer--try-render t)
   (remove-hook 'window-state-change-hook #'blamer--try-render t))
 
+(defun blamer--get-user-name ()
+  "Query git config for user name"
+  (let ((command-output (apply #'vc-git--run-command-string nil blamer--git-author-cmd)))
+    (if command-output
+        (replace-regexp-in-string "\n\\'" "" command-output))))
+
```

Comment:
I think it's a good place to replace `let` to `when-let`, in this case we can remove nested `if` conditional

Reply:
You are right of course, should be fixed now.